src/pages/SearchHelpPage.vue

```diff
@@ -0,0 +1,195 @@
+
+    <h2>Search by tags</h2>
+    <p>
+      Put <em>tags.name:</em> in front of your search term to see only results that are
+      tagged with the specified tag. For example,
+    <a href='https://search.creativecommons.org/search?q=tags.name%3A"sky"'>
+      <em>tags.name:"sky"</em>
+    </a>
+    <br />
+    If you want to combine tags, you can use the AND operator, or the OR operator, depending on
+    your search preference.
+
+    <br />
+
+    <p>Example:
+    <a href='https://search.creativecommons.org/search?q=tags.name%3A"sky"%20AND%20tags.name%3A"airplane"'>
+      <em>tags.name:"sky" AND tags.name:"airplane"</em>
+    </a>
+    <br />
+    This will search for images with both tags, sky and airplane.
+    </p>
+
+    <p>Example:
+    <a href='https://search.creativecommons.org/search?q=tags.name%3A%20"phone"%20OR%20tags.name%3A%20"sky"'>
+      <em>tags.name: "phone" OR tags.name: "sky"</em>
+    </a>
+    <br />
+    This will search for images tagged with sky or with phone, but not necessarily both.
+    </p>
+
+    <h2>Combining keywords</h2>
+
+    <p>
+      Put <em>AND</em> in between two queries to see only results that match both queries.
+      For example:
+      <ul>
+        <li>
+          <a href='https://ccsearch.creativecommons.org/search?q=title%3A%20"nature"%20AND%20provider%3A%20"met"'>
+            <em>title: "nature" AND provider: "met"</em>
+          </a>
+          will return images from the MET and whose title contains nature.
+        </li>
+        <li>
+          <a href='https://ccsearch.creativecommons.org/search?q=tags.name%3A"sky"AND%20tags.name%3A"airplane"'>
+            <em>tags.name:"sky"AND tags.name:"airplane"</em>
+          </a>
+          will return images tagged both sky and  airplane.
+        </li>
+      </ul>
+    </p>
+
+    <p>
+      Put <em>OR</em> in between two queries to see only results that match either query.
+      For example:
+      <ul>
+        <li>
+          <a href='https://ccsearch.creativecommons.org/search?q=title%3A%20"nature"%20AND%20provider%3A%20"met"'>
+            <em>provider:"met" OR provider:"flickr"</em>
+          </a>
+          will return images which are either from MET or from Flickr.
+        </li>
+        <li>
+          <a href='https://ccsearch.creativecommons.org/search?q=tags.name%3A"phone"%20OR%20tags.name%3A"tablet"'>
+            <em>tags.name:"phone" OR tags.name:"tablet"</em>
+          </a>
+          will return images tagged either phone or  tablet.
+        </li>
+      </ul>
+    </p>
+
+    <p>
+      Put <em>NOT</em> in between two queries to see only results that match the first query
+      and do not match the second query. You can also use parentheses to combine more than one
+      of these operations For example:
+      <ul>
+        <li>
+          <a href='https://ccsearch.creativecommons.org/search?q=license%3A"by"%20AND%20%28NOT%20provider%3A"flickr"%29'>
+            <em>license:"by" AND (NOT provider:"flickr")</em>
```

Comment:
This does look cleaner but [Lucene's syntax guide](https://lucene.apache.org/core/2_9_4/queryparsersyntax.html) says:

> Note: The NOT operator cannot be used with just one term. For example, the following search will return no results:
>
> `NOT "jakarta apache"`

I'm not sure these examples will work, could you double check and update if needed?

Reply:
This works. The `NOT provider: "flickr"` in the end filters out items from Flickr